<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MazeView</class>
 <widget class="QMainWindow" name="MazeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MazeView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #505050;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>910</width>
      <height>660</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: #505050;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabMaze">
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #505050;
}</string>
     </property>
     <attribute name="title">
      <string>Maze</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>610</width>
        <height>610</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGraphicsView {
	border: 2px solid gray;
	background-color: #303030;
}</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>270</width>
        <height>610</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
	border: 2px solid gray;
	background-color: #303030;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnOpenFile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>250</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #8F8F8F;
	border-radius: 10px;
	border: 3px solid gray;
	color: #82D625;
}

QPushButton:hover {
    border: 3px solid #82D625;
}

QPushButton:pressed {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSaveFile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>250</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #8F8F8F;
	border-radius: 10px;
	border: 3px solid gray;
	color: #82D625;
}

QPushButton:hover {
    border: 3px solid #82D625;
}

QPushButton:pressed {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Save File</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="RowsValue">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>150</y>
         <width>145</width>
         <height>25</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox {
  border: 2px solid rgb(76,76,76);
  border-radius: 5px;
  background-color: #8F8F8F;
  color: rgb(255,255,255);
}

QSpinBox::down-button  {
  subcontrol-origin: margin;
  subcontrol-position: center left;
  image: url(:/images/left_arrow.png);
  background-color: rgb(76,76,76);
  left: 2px;
  height: 22px;
  width: 12px;
}

QSpinBox::up-button  {
  subcontrol-origin: margin;
  subcontrol-position: center right;
  image: url(:/images/right_arrow.png);
  background-color: rgb(76,76,76);
  right: 2px;
  height: 22px;
  width:12px;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="ColsValue">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>185</y>
         <width>145</width>
         <height>25</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox {
  border: 2px solid rgb(76,76,76);
  border-radius: 5px;
  background-color: #8F8F8F;
  color: rgb(255,255,255);
}

QSpinBox::down-button  {
  subcontrol-origin: margin;
  subcontrol-position: center left;
  image: url(:/images/left_arrow.png);
  background-color: rgb(76,76,76);
  left: 2px;
  height: 22px;
  width: 12px;
}

QSpinBox::up-button  {
  subcontrol-origin: margin;
  subcontrol-position: center right;
  image: url(:/images/right_arrow.png);
  background-color: rgb(76,76,76);
  right: 2px;
  height: 22px;
  width:12px;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGenerate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>250</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #8F8F8F;
	border-radius: 10px;
	border: 3px solid gray;
	color: #82D625;
}

QPushButton:hover {
    border: 3px solid #82D625;
}

QPushButton:pressed {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>95</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: #82D625;
}</string>
       </property>
       <property name="text">
        <string>MAZE ROWS:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>185</y>
         <width>95</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: #82D625;
}</string>
       </property>
       <property name="text">
        <string>MAZE COLS:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCave">
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #505050;
}</string>
     </property>
     <attribute name="title">
      <string>Cave</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
